api/WikiClient.ts

```diff
@@ -14,6 +15,17 @@ export type WikiListResponse = {
 
 export default class WikiClient extends BaseClient {
   public async getEntries(): Promise<WikiEntry[]> {
-    return [...(await this.get<WikiListResponse>('/wiki')).entries, longExampleEntry];
+    let result: WikiListResponse = {
+      entry_count: 0,
+      entries: [],
+    };
+    await this.get<WikiListResponse>('/wiki')
+      .then((response) => {
+        result = response;
+      })
+      .catch(() => {
+        Alert.alert('Keine Verbindung', 'Leider besteht zurzeit keine Verbindung zu unserem Server');
+      });
+    return [...result.entries, longExampleEntry];
```

Comment:
Das `then` hier ist unnötig, im `catch` kannst du direkt einen Ersatzwert zurückgeben:

```suggestion
    const result = await this.get<WikiListResponse>('/wiki').catch(() => {
      Alert.alert('Keine Verbindung', 'Leider besteht zurzeit keine Verbindung zu unserem Server');
      return {
        entry_count: 0,
        entries: [],
      };
    });
    return [...result.entries, longExampleEntry];
```

Oder alternativ mit try/catch:
```suggestion
    try {
      return [...(await this.get<WikiListResponse>('/wiki')).entries, longExampleEntry];
    } catch (e) {
      console.error(e);
      return [];
    }
```

Im `SecurityNetClient` natürlich genauso